js/src/offcanvasmenu.js

```diff
@@ -0,0 +1,393 @@
+/**
+ * --------------------------------------------------------------------------
+ * Arizona Bootstrap: offcanvasmenu.js
+ * Licensed under MIT (https://github.com/az-digital/arizona-bootstrap/blob/master/LICENSE)
+ * --------------------------------------------------------------------------
+ */
+
+import $ from 'jquery'
+import Util from '../../node_modules/bootstrap/js/src/util.js'
+import { version } from '../../package.json'
+
+/**
+ * ------------------------------------------------------------------------
+ * Constants
+ * ------------------------------------------------------------------------
+ */
+
+const NAME                = 'offcanvasmenu'
+const AZ_VERSION          = `${version}`
+const DATA_KEY            = 'az.offcanvasmenu'
+const EVENT_KEY           = `.${DATA_KEY}`
+const DATA_API_KEY        = '.data-api'
+const JQUERY_NO_CONFLICT  = $.fn[NAME]
+
+const Default = {
+  toggle : true,
+  parent : ''
+}
+
+const DefaultType = {
+  toggle : 'boolean',
+  parent : '(string|element)'
+}
+
+const EVENT_OPEN           = `open${EVENT_KEY}`
+const EVENT_OPENED         = `opened${EVENT_KEY}`
+const EVENT_CLOSE          = `close${EVENT_KEY}`
+const EVENT_CLOSED         = `closed${EVENT_KEY}`
+const TA_API = `click${EVENT_KEY}${DATA_API_KEY}`
```

Comment:
Was this variable name intentional?  (Just wondering if it should be `DATA_API`?)